rotkehlchen/data_import/importers/blockpit.py

```diff
@@ -0,0 +1,216 @@
+                amount=amount_in,
+                rate=rate,
+                fee=fee_amount,
+                fee_currency=fee_currency,
+                notes=notes,
+            )
+            self.add_trade(write_cursor, trade)
+
+        elif transaction_type in {'Deposit', 'Withdrawal', 'NonTaxableIn', 'NonTaxableOut'}:
+            if transaction_type in {'Deposit', 'NonTaxableIn'}:
+                direction = 'Incoming'
+                category = AssetMovementCategory.DEPOSIT
+            else:
+                direction = 'Outgoing'
+                category = AssetMovementCategory.WITHDRAWAL
+            asset_movement = AssetMovement(
+                location=location,
+                category=category,
+                address=None,
+                transaction_id=None,
+                timestamp=timestamp,
+                asset=asset_resolver(csv_row[f'{direction} Asset']),
+                amount=deserialize_asset_amount(csv_row[f'{direction} Amount']),
+                fee=fee_amount,
+                fee_asset=fee_currency,
+                link='',
+            )
+            self.add_asset_movement(write_cursor, asset_movement)
+
+        elif transaction_type in {
+            'Airdrop',
+            'Bounties',
+            'Staking',
+            'Lending',
+            'Fee',
+            'Gift',
+            'Margin_trading_loss',
+            'Margin_trading_profit',
+        }:
+            timestamp_ms = ts_sec_to_ms(timestamp)
+
+            if transaction_type in {'Fee', 'Gift', 'Margin_trading_loss'}:
+                asset = asset_resolver(csv_row['Outgoing Asset'])
+                amount = deserialize_asset_amount(csv_row['Outgoing Amount'])
+                event_type = HistoryEventType.SPEND
+                event_subtype = HistoryEventSubType.NONE
+                event_description = 'Spend'
+            else:
+                asset = asset_resolver(csv_row['Incoming Asset'])
+                amount = deserialize_asset_amount(csv_row['Incoming Amount'])
+                event_type = HistoryEventType.RECEIVE
+                event_subtype = HistoryEventSubType.NONE
+                event_description = 'Receive'
+
+            if transaction_type == 'Staking':
+                event_type = HistoryEventType.STAKING
+                event_subtype = HistoryEventSubType.REWARD
+                event_description = 'Staking reward of'
+            elif transaction_type == 'Fee':
+                event_subtype = HistoryEventSubType.FEE
+                event_description = 'Fee of'
+
+            event = HistoryEvent(
+                event_identifier=f'{BLOCKPIT_EVENT_PREFIX}_{uuid4().hex}',
+                sequence_index=1,
+                timestamp=timestamp_ms,
+                location=location,
+                event_type=event_type,
+                event_subtype=event_subtype,
+                asset=asset,
+                balance=Balance(amount, ZERO),
```

Comment:
zero is default, for usd value. Please don't specify it. Also if you were to specify both you should have used kwargs.

Both here and in the rest of the places you use this.

```suggestion
                balance=Balance(amount),
```